manticore/native/memory.py

```diff
@@ -892,6 +892,28 @@ def perms(self, index):
         else:
             return self.map_containing(index).perms
 
+    def max_access_size(self, addr_start, max_size, access):
+        """
+        Finds maximum executable memory size
```

Comment:
> Finds maximum executable memory size

Since we don't reuse the function for other permissions yet and I am unsure if it is needed for now (since `.access_ok` accepts a slice too) we should probably go back to `max_exec_size`? :P